api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -586,4 +586,30 @@ public Locale getLocale() {
 	public void setLocale(Locale locale) {
 		this.locale = locale;
 	}
+
+	/**
+	 * @return the value of the Global Propriety Handle Timezones 
+	 */
+	public boolean handleTimeZones(){
```

Comment:
@mogoodrich are you ok with this name?
Or perhaps `hasTimeZonesSupport()` or `timeZonesSupport()`?

Would be good to know early because this will be referred to in many modules.

@ibacher you're great at this kind of thing ;-) Your ideas would be more than welcome.

Reply:
I know we've talked about this ad nauseum but  I always forget the exact details... "handle" time zones feels a bit vague... I'd be in favor of something more specific... per my above comment are we says "never format timezones for display on the server side, force RFC3339 format"  (not my suggestion for a GP.. :) )

Reply:
Maybe something like: `sendDatetimeAsUTC`? Not my favourite phrasing, but accurate.

Reply:
That's decent @ibacher ....